### PR TITLE
Bo Lee

### PR DESCRIPTION
**Pull request by boestherlee**
@kaszklar can you please review my file?

### CODE REVIEW

File: Bo Lee.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# #Bo Lee - Orange
```
Comment (kaszklar) [inline Bo Lee.py:4]: But what is your favorite QMSS class?

**Comment by amorris41291** [comment]
@boestherlee orange is one of the top 12 colors!

**Review by kaszklar — COMMENTED** [review]
How about now?

File: Bo Lee.py (hunk 1 above)
Comment (boestherlee) [reply Bo Lee.py:4]: Hmm that's a difficult question to answer, I need some more time to get back to you on that

**Pull request closed without merging**